CMP1903M Assessment 1 Base Code/Report.cs

```diff
@@ -11,14 +11,20 @@ class Report
         //Handles the reporting of the analysis
         //Maybe have different methods for different formats of output?
         //eg.   public void outputConsole(List<int>)
-        public static void Reports(List <object> measurements)        
+        public static void Reports(Analyse analyse)        
         {
-            var m = measurements;
+            var m = analyse.measurements;
             Console.WriteLine("Sentences: {0}\nVowels: {1}\nConsonants: {2}\nUpperCase Letters: {3}\nLowerCase Letters: {4}\nMost common character: {5}\n"
             ,m[0], m[1], m[2], m[3],m[4],m[5]);
```

Comment:
This line must burn. 🔥🔥🔥
Let's clean this up a bit.
```suggestion
            Console.WriteLine($"Sentences: {m[0]}\n" +
                              $"Vowels: {m[1]}\n" +
                              $"Consonants: {m[2]}\n" +
                              $"Uppercase Letters: {m[3]}\n" +
                              $"Lowercase Letters: {m[4]}\n" +
                              $"Most common character: {m[5]}\n"
```
For extra points, make `analyse.measurements` a `Dictionary<string, int>` so the indexes are more legible:
```csharp
            Console.WriteLine($"Sentences: {m["sentences"]}\n" +
                              $"Vowels: {m["vowels"]}\n" +
                              $"Consonants: {m["consonants"]}\n" +
                              $"Uppercase Letters: {m["upperCaseLetters"]}\n" +
                              $"Lowercase Letters: {m["lowerCaseLetters"]}\n" +
                              $"Most common character: {m["commonCharacters"]}\n"
```

Reply:
Changes made